rwengine/src/engine/GameState.cpp

```diff
@@ -94,3 +96,85 @@ void GameState::showHelpMessage(const GameStringKey& id) {
     text.addText<ScreenTextType::Help>(
             ScreenTextEntry::makeHelp(id, world->data->texts.text(id)));
 }
+
+Rampage::Status Rampage::getStatus() const {
+    return status;
+}
+
+void Rampage::init(const std::string& gxtEntry, const int32_t weaponID,
+                   const int32_t time, const int32_t kills,
+                   const std::array<int32_t, 4>& modelsToKill,
+                   const bool headshot) {
+    // TODO: Document why
+    if (gxtEntry != "PAGE_00") {
```

Comment:
There are 2 missions (Trial By Fire, Uzi Money) that use ```init_rampage``` opcode for their own purposes. They pass custom onscreen message, and there is a check if message isn't default then it's special mission rampage initialization, so don't increment rampagesPassed stat, because that's not a real rampage.

Reply:
Good to know, thank you - going to document that appropriately.
Nice to still see you around!